src/index.html

```diff
@@ -10,8 +10,150 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
+  <body> 
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method=post">
+      <fieldset>
+        <legend> 
+          Personal information
+        </legend>
+      <label class="form-field">
+        Surname:
+        <input 
+          type="text" 
+          name="sname"
+          id="sname" 
+          autocomplete="off" 
+          required>
```

Comment:
```suggestion
        <input 
          type="text" 
          name="sname"
          id="sname" 
          autocomplete="off" 
          required
        >
```
align closing brace like that